styleguide/derek/_partials/solutions/additional-resources.ejs

```diff
@@ -1,48 +1,70 @@
+<%
+// While seemingly redundant, this is actually necessary for EJS to evaluate variable scope.
+var isSubpanel = !!isSubpanel;
```

Comment:
So apparently ejs is particular about variable scope, which makes this line necessary. Without it, the `isSubpanel` value wasn't carrying through. 

As far as an explanation as to what this is, this is basically how we can identify if we are in a subpanel or not. This allows us to reuse as much of the existing patterns as possible, and just create exceptions for subpanels instead of having two separate bits of code to maintain for ever single pattern.